test/unit/test-validator-integration.js

```diff
@@ -71,7 +71,10 @@ describes.fakeWin('validator-integration', {}, (env) => {
         .stub(eventHelper, 'loadPromise')
         .returns(Promise.resolve());
 
-      loadScript(win.document, 'http://example.com');
+      loadScript(
+        win.document,
+        'https://cdn.ampproject.org/v0/validator_wasm.js'
```

Comment:
@amaltas do you happen to know if there are any AMP Cache that implements/hosts the validator_wasm file. I am currently assuming that it is only us on the Google AMP Cache